src/index.html

```diff
@@ -1,17 +1,201 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      onsubmit="onSubmit()"
+    >
+
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <div class="form_field">
+          <label for="surname">
+          Surname:
+          </label>
+          <input
+          type="text"
+          name="surname"
+          required
+          id="surname">
+        </div>
+
+        <div class="form_field">
+          <label for="name">
+          Name:
+          </label>
+          <input
+          type="text"
+          name="name"
+          required
+          id="name">
+        </div>
+
+        <div class="form_field">
+          <label for="how old are You">
+          How old are You?
+          </label>
+          <input
+          type="number"
+          name="how old are You?"
```

Comment:
Use shorter words for name. Also, name and id should not contain spaces.
```suggestion
          name="age"
```

Reply:
Done